uvicorn/_types.py

```diff
@@ -0,0 +1,193 @@
+import socket
+import sys
+from asyncio import Transport
+from os import PathLike
+from typing import (
+    TYPE_CHECKING,
+    Awaitable,
+    Callable,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+    Union,
+)
+
+if sys.version_info < (3, 8):
+    if TYPE_CHECKING:
+        from typing_extensions import Literal, TypedDict
+    else:  # due to different mypy warnings raised during CI for python 3.7 and 3.8
```

Comment:
This looks a bit off. Are you able to make the topmost try/except pass mypy by adding `# type: ignore` for any of the imports?

Reply:
I dont understand what you mean :)

Reply:
well maybe like this, not sure that's what you meant !